modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -83,47 +48,81 @@ dynamicPlugins:
+          parent: favorites # <3>
+        favorites: # <4>
+          icon: favorite # <5>
+          title: Favorites # <6>
+          priority: 100 # <7>
 ----
---
+  <1> `my-plugin`: Matches `path` in `dynamicRoutes`
+  <2> `priority`: Controls order of plugins under the parent menu item.
+  <3> `parent`: Nests this plugin under the `favorites` parent menu item.
+  <4> `favourites`: Configuration for the parent menu item.
+  <5> `icon`: Displays the `favorite` icon from the {product-very-short} system icons.
+  <6> `title`: Displays the title name for the parent menu item.
+  <7> `priority`: Order of the `favourites` menu item in the sidebar.
 
-. To ensure that a menu item is identified as a main menu item, you must add the `default.` prefix to its key. For example:
-+
---
-.Example configuration of main menu items in sidebar navigation
+
+. To modify a main menu item or add a custom menu item, add a section to the `default.main-menu-items` > `menuItems` section in your `{my-app-config-file}` file. Use the `default.` prefix to identify the key as a main menu item.
++ 
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    default.main-menu-items: # key for configuring static main menu items
-      menuItems: 
-        default.<menu_item_name>: # key of the menu item configuration. `default.` prefix is required for a main menu item key <1>
-          parent: my_menu_group # optional, specifies the parent menu item for this item
-          priority: 10 # optional, specifies the order of this menu item within its menu level
-        default.<menu_group_parent_item_name>: # must be configured if it is specified as the parent of any menu items. `default.` prefix is required for a main group parent item key <1>
-          icon: my_menu_group_icon # required for parent menu items, defines the icon for the menu group
-          title: my_menu_group_title # required for parent menu items, defines the icon for the menu group
-          priority: 100 # optional, specifies the order of the parent menu item in the sidebar
+    default.main-menu-items:
+      menuItems:
+        default._<menu_group_parent_item_name>_: # <1>
+          icon: # home | group | category | extension | school | _<my_icon>_ # <2>
+          title: _<menu_group_parent_title>_ # <3>
+          priority: 10 # <4>
+        default._<menu_item_name>_: # <5>
+          parent: _<menu_group_parent_item_name>_ # <6>
+          icon:  # home | group | category | extension | school | _<my_icon>_ # <7>
+          title: _<my_menu_title>_ # <8>
+          to: _<path_to_the_menu_target_page>_ # <9>
+          priority: 100 # <10>
 ----
+  <1> `default._<menu_group_parent_item_name>_`: Enter the menu group parent item name to configure static main menu items.
+  <2> `icon`: Enter the menu icon. Required for parent menu items.
+  <3> `title`: Enter the menu group title. Required for parent menu items. To hide the title from the sidebar, set the title as an (`" "`) empty string.
```

Comment:
```suggestion
  <3> `title`: Enter the menu group title. Required for parent menu items.
```

Setting an empty string is only for hiding a default static menu item, not for menu group parent item.